app/Providers/AppServiceProvider.php

```diff
@@ -77,5 +78,6 @@ public function boot(): void
     public function register(): void
     {
         $this->app->alias('bugsnag.multi', \Psr\Log\LoggerInterface::class);
+        Passport::ignoreMigrations();
```

Comment:
They aren't, afaik

Reply:
So why copy them into the repo...

Reply:
at one point we did need it for adding a field to signify first-party clients. that's no longer a thing but the last time I tried to remove the migrations didn't go smoothly locally.  i'll see what I can do

Reply:
I thought they should just get loaded out of the package. That's how the Nova and Horizon ones work, I think?

Reply:
If you use UUIDs for client IDs in Passport (which prevents easily guessing client IDs -- not that client IDs are intended to be secret -- things you can do by incrementing/decrementing an integer client ID by one, and client IDs revealing how many clients there are), it [automatically publishes the Passport migrations](https://github.com/laravel/passport/blob/10.x/src/Console/InstallCommand.php#L51) and replaces some text to use UUIDs as the primary key in the relevant spots.  Since it's publishing the migrations, I think it makes sense to ignore the Passport migrations.

Additionally, when Passport [makes changes that require updating the schema](https://github.com/laravel/passport/blob/10.x/UPGRADE.md#support-for-multiple-guards), they just tell you to add columns or whatever to the Passport tables, so either way you have to add migrations yourself 🙄

Reply:
Got it, makes sense.